mkdocs.yml

```diff
@@ -4,49 +4,63 @@
 #
 site_name: Deis Documentation
 pages:
-  - Welcome: index.md
-  - Quick Start: installing-workflow/quickstart.md
+  - Home: index.md
+  - Quick Start:
+    - Overview: quickstart/index.md
+    - Install CLI Tools: quickstart/install-cli-tools.md
+    - AWS:
+      - Boot: quickstart/provider/aws/boot.md
+      - DNS: quickstart/provider/aws/dns.md
+      - Install Workflow: quickstart/provider/aws/install-aws.md
+    - GKE:
+      - Boot: quickstart/provider/gke/boot.md
+      - DNS: quickstart/provider/gke/dns.md
+      - Install Workflow: quickstart/provider/gke/install-gke.md
+    - Vagrant:
+      - Boot: quickstart/provider/vagrant/boot.md
+      - DNS: quickstart/provider/vagrant/dns.md
+      - Install Workflow: quickstart/provider/vagrant/install-vagrant.md
+    - Deploy Your First App: quickstart/deploy-an-app.md
   - Understanding Workflow:
     - Concepts: understanding-workflow/concepts.md
     - Architecture: understanding-workflow/architecture.md
     - Components: understanding-workflow/components.md
   - Installing Workflow:
-    - Quick Start: installing-workflow/quickstart.md
+    - Installing Workflow: installing-workflow/index.md
     - System Requirements: installing-workflow/system-requirements.md
-    - Installing Workflow: installing-workflow/installing-deis-workflow.md
     - Configuring Object Storage: installing-workflow/configuring-object-storage.md
   - Using Workflow:
-    - Installing the Client: using-workflow/installing-the-client.md
-    - Registering a User: using-workflow/registering-a-user.md
-    - Deploying an Application: using-workflow/deploying-an-application.md
-    - Process Types and the Procfile: using-workflow/process-types-and-the-procfile.md
-    - Using Buildpacks: using-workflow/using-buildpacks.md
-    - Using Dockerfiles: using-workflow/using-dockerfiles.md
-    - Using Docker Images: using-workflow/using-docker-images.md
-    - Configuring an Application: using-workflow/configuring-an-application.md
-    - Managing an Application: using-workflow/managing-an-application.md
-    - Application SSL Certificates: using-workflow/application-ssl-certificates.md
+    - Command Line Interface: using-workflow/cli.md
```

Comment:
this is a confusing title choice, since "users and registration" and "user ssh keys" both use the command line interface. Is there any other way we would use Workflow other than through the CLI?